src/Explorer/Tabs/SettingsTab.ts

```diff
@@ -1233,6 +1233,9 @@ export default class SettingsTab extends TabsBase implements ViewModels.WaitsFor
   };
 
   public onRevertClick = (): Q.Promise<any> => {
+    TelemetryProcessor.trace(Action.DiscardSettings, ActionModifiers.Mark, {
+      message: "Settings Discarded"
```

Comment:
Not sure whether this is needed, since the Action already describes. We can save a few bytes.